searchobject/src/main/java/org/jalau/at18/searchobject/model/facedetector/FaceDetect.java

```diff
@@ -0,0 +1,138 @@
+package org.jalau.at18.searchobject.model.facedetector;
+/**
+ * Copyright (c) 2022 Jala University.
+ *
+ * This software is the confidential and proprieraty information of Jalasoft
+ * ("Confidential Information"). You shall not disclose such Confidential
+ * Information and shall use it only in accordance with the terms of the
+ * Licence agreement you entered into with Jalasoft
+ */
+
+
+import org.jalau.at18.searchobject.common.logger.At18Logger;
+import org.opencv.core.Mat;
+import org.opencv.core.MatOfRect;
+import org.opencv.core.Point;
+import org.opencv.core.Rect;
+import org.opencv.core.Scalar;
+import org.opencv.imgcodecs.Imgcodecs;
+import org.opencv.imgproc.Imgproc;
+import org.opencv.objdetect.CascadeClassifier;
+
+import java.util.logging.Logger;
+
+
+/**
+ * It is responsible for analyze an image and detect if there it's a face or not.
+ *
+ * @author Daniela Martinez
+ * @version 1.0
+ */
+public class FaceDetect {
+    private String faceDetection;
+    private static String current_dir; //full image path direction
+    private static String result; //result of the face detection in a multiple or profile image
+
+    private static int face_Detect;
+
+    /**
+
+     * Constructor to the face detection
+
+     * @param file the image that it's going to be in the process of detection
+     * @param type the tipe of result we desire to find in the input image
+
+     */
+    public FaceDetect (String file, String type) {
+        Logger log = At18Logger.getLogger();
+        faceDetection (file, type);
+
+    }
+    /**
+
+     * Method that it's the process of the face detection where create the square in the face if it's detected
+
+     */
+
+    public static void faceDetection (String file, String type) {
+        Logger log = At18Logger.getLogger();
+        //Current_dir will take the direction of the path
+        current_dir = System.getProperty("user.dir");
+
+        //run the dll file, so the Open CV works correctly
+        System.load(current_dir + "\\src\\main\\resources\\libraries\\opencv\\x64\\opencv_java454.dll");
+
+        //imgFile it's the file we want to analyze if there it's a person in the photo
+        String imgFile = current_dir +"\\"+ file;
+
+        //Decodes the images
+        Mat src = Imgcodecs.imread(imgFile.toString());
```

Comment:
the method .toString is not necessary in this case, because imgFile is already a String.

Reply:
yes, it's not necessary, I already take out the toString()